example/code/glob_avg.py

```diff
@@ -0,0 +1,55 @@
+'''
+Creates a monthly temparature time series for a global average
+temperature, using of all Berkeley region data per time index. The
+average is of the temperature absolute value, the anomaly plus the
+base value for a time/region.  Missing data is interpolated using
+Pandas interpolate(). Data is from http://berkeleyearth.org/data/, see
+"Monthly Land" under the Gridded section, and use the "Equal Area"
```

Comment:
So, this is just the average land temperature, right? Should clarify that at the top.

Reply:
Done